src/styles/mixins.less

```diff
@@ -90,3 +90,457 @@
 	-webkit-transform:translate3d(0,0,0);
 }
 
+
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+#Normalize {
```

Comment:
I'd suggest we keep this lowercase `#normalize` so it doesn't make people think Normalize is a react component (our existing convention).